packages/typespec-ts/src/modular/buildRootIndex.ts

```diff
@@ -75,6 +75,26 @@ export function buildRootIndex(
   exportFileContentsType(context, rootIndexFile);
 }
 
+export function exportModelsIfAbsent(
```

Comment:
why this is not working https://github.com/Azure/autorest.typescript/pull/3222/files#diff-d137486a43daabc048002aa2a9c10a3506f9abd6c3ec2dc63904e4b5d6dbd147L58?